### PR TITLE
Add solution for anomaly detection

### PR DESCRIPTION
**Pull request by dubansal**
Add solution for anomaly detection using spike and dip.

### CODE REVIEW

File: eventhubs-streamanalytics-eventhubs/anomalydetection/create-solution.sh [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+    exit 1
+}
+
+trap 'on_error $LINENO' ERR
+
+export PREFIX=$1
+export RESOURCE_GROUP=$PREFIX
+export LOCATION=eastus
+
+# 10000 messages/sec
```
Comment (chetanmsft) [inline eventhubs-streamanalytics-eventhubs/anomalydetection/create-solution.sh]: Need to delete these from here and parent create-solution.sh as well, its causing confusion.

File: eventhubs-streamanalytics-eventhubs/anomalydetection/create-solution.sh [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+if [ -z $STEPS ]; then  
+    export STEPS="CIDPT"    
+fi
+
+# remove log.txt if exists
+rm -f log.txt
+
+echo
+echo "Streaming at Scale with Stream Analytics and Event Hubs"
```
Comment (chetanmsft) [inline eventhubs-streamanalytics-eventhubs/anomalydetection/create-solution.sh]: with Anomaly Detection scenario

File: eventhubs-streamanalytics-eventhubs/anomalydetection/create-solution.sh (hunk 1 above)
Comment (dubansal) [reply eventhubs-streamanalytics-eventhubs/anomalydetection/create-solution.sh]: I think this is helpful to leave here. Did you get feedback that it causes confusion?

Comment (dubansal) [reply eventhubs-streamanalytics-eventhubs/anomalydetection/create-solution.sh]: RESOURCE_GROUP  is actually used.

Comment (chetanmsft) [reply eventhubs-streamanalytics-eventhubs/anomalydetection/create-solution.sh]: I meant the duplicate "# 10000 messages/sec" section with different numbers is creating confusion. Users wont know which one to uncomment to actually repro the scenario.

Comment (dubansal) [reply eventhubs-streamanalytics-eventhubs/anomalydetection/create-solution.sh]: Removed the duplicate 10000 messages/sec section

**Pull request merged**